.vscode/settings.json

```diff
@@ -19,7 +19,7 @@
     "json"
   ],
   "editor.codeActionsOnSave": {
-    "source.fixAll.eslint": true
+    "source.fixAll.eslint": "explicit"
```

Comment:
What's the reason for this?

Reply:
VSCode automatically updated it after I updated to the newest version of VSCode.